book.toml

```
@@ -1,7 +1,7 @@
 [book]
 title = "예제로 배우는 러스트 (Rust by Example) 한국어판"
-description = "A description"
-author = "The Rust Community"
+description = "예제로 배우는 러스트 (Rust by Example)의 한국어판입니다."
+author = "러스트 한국 커뮤니티"
```

https://github.com/rust-kr/ org의 이름과 일치시키는 편이 좋겠습니다.

```suggestion
author = "한국 러스트 사용자 그룹"
```